src/utils/osUtils.ts

```diff
@@ -43,3 +43,26 @@ export function isArm64Mac(): boolean {
 export function isLinux(): boolean {
     return os.platform() !== 'win32' && os.platform() !== 'darwin';
 }
+
+export function getNativeArchitecture(): string {
```

Comment:
This is specifically converting the native architecture to a Docker/Go architecture value and should probably live under DockerClientBase.

Reply:
yeah good point, this will avoid confusion